README.md

```diff
@@ -5,9 +5,6 @@
 Services through Snowflake CLI, you must register for its PrPr. For more information, you can contact a
 Snowflake sales representative.
 
-For complete installation and usage instructions, refer to the
```

Comment:
Will we add a new link after the 2.0 release?

Reply:
No, we shouldn't link to docs from repo unless we are in PuPr